go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/ssvlabs/ssv
 
-go 1.22
+go 1.22.0
+
+toolchain go1.23.1
```

Comment:
This change is the result of running `go mod tidy` after adding `tailscale.com/util/singleflight` (note also, I'm not using the latest version of this package because it will require to bump the `go 1.22` line up to `go 1.23.1`),

I'm new to this `toolchain` concept, from what I've gathered it's used as "recommended" version (it will be downloaded and used in case you are trying to build with lower version).

Reply:
Removed `toolchain` directive, see https://github.com/ssvlabs/ssv/pull/1840#discussion_r1834748874 for details.